modules/installation-configuration-parameters.adoc

```diff
@@ -388,6 +388,10 @@ control plane machine pool.
 belong to the same region as the cluster. This is required for regions that require a custom {op-system} AMI.
 |Any published or custom {op-system} AMI that belongs to the set AWS region.
 
+|`platform.aws.hostedZone`
+|An existing Route 53 private hosted zone for the cluster. Providing an existing hosted zone allows you to manually create DNS records for your cluster. You can only use a pre-existing hosted zone when also supplying your own VPC. The hosted zone must already be associated with the user-provided VPC before installation. If undefined, the installation program creates a new hosted zone.
```

Comment:
I am not sure about the following sentence. Even when supplying a hosted zone, the installer and in-cluster operators will still create DNS records in the hosted zone.
> Providing an existing hosted zone allows you to manually create DNS records for your cluster.

Reply:
Makes sense; I'll remove that sentence :+1: